src/public/sass/UI/header/_header.scss

```
@@ -1,37 +1,101 @@
-@use "base/variables" as v;
-@use "base/mixins" as m;
+@use '../../base/variables' as v;
+@use '../../base/mixins' as m;
```

```suggestion
@use 'sass:color';
@use '../../base/variables' as v;
@use '../../base/mixins' as m;
```


@CarlosMay7 the problem that you had is because you were not importing the sass:color utils

https://sass-lang.com/documentation/modules/
https://sass-lang.com/documentation/modules/color/